test/adapters/cuda/kernel_tests.cpp

```diff
@@ -74,9 +74,9 @@ const char *threeParamsTwoLocal = "\n\
 TEST_P(cudaKernelTest, CreateProgramAndKernel) {
 
     uur::raii::Program program = nullptr;
-    ASSERT_SUCCESS(urProgramCreateWithBinary(context, device, sizeof(ptxSource),
-                                             (const uint8_t *)ptxSource,
-                                             nullptr, program.ptr()));
+    ASSERT_SUCCESS(urProgramCreateWithBinary(
```

Comment:
heh, pls remove Benie's commit from this PR

Reply:
Done